### DISCUSSION TITLE
Interaction-prompt=none - is it working for you?

### DISCUSSION THREAD

**Discussion by VashUAR** [google/model-viewer#4671] [Q&A] [answered]
Hi everyone.

RE: interaction-prompt="none"

My understanding of the attribute is used to prevent the 3D model from automatically rendering on the ground as soon as the ground is found.  Is that correct?

If it is, then either it is not working or I am doing something wrong.

As I go to View in your space- the UI indicates to move the point the phone at the ground and move the camera around.
Still the 3D model automatically renders on the ground as soon as it finds the ground plane.
At times the model is placed in an undesired space while the phone is moved around as part of the scanning the ground.

I am testing on both Safari iOS and Chrome Android.
Using 

		<script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
		<model-viewer 
			src="chair.glb" 
			poster="chair.jpg" 
			interaction-prompt="none"
			... >
		  <button slot="ar-button" id="ar-button">
			View in your space
		  </button>
		  </model-viewer>
		  
Can anyone point me in the right direction please?
I want to give the user a chance to tap on the screen to place model down.

Thank you.

**Comment by danilokrulj** [comment, marked as answer]
I think you are not correct. From what I understand interaction-prompt="none" just means you wont have the little hand showing you you can move the model (in the viewer, before entering AR). If you don't have interaction-prompt="none" you will see after a few seconds of not moving that there is a little hand visual showing you you can rotate the model, kind of moves it left to right a bit.

**Reply by VashUAR** [reply]
Thank you, danilokrulj.
